src/main/scala/cognite/spark/v1/RawTableRelation.scala

```diff
@@ -298,12 +298,14 @@ class RawTableRelation(
   }
 }
 
-object RawTableRelation extends NamedRelation {
+object RawTableRelation extends NamedRelation with UpsertSchema {
```

Comment:
also should have ReadSchema?

Reply:
Readschema is not in use. I kept it where it is currently but didn't add it anywhere.

Since there is no val ReadSchema in RawTableRelation I didn't add it. I think by default we use abortSchema when reading but I haven't checked, might be upsertSchema as well.

When I add file content schema that might become relevant.